Precision about HTTP/2 handling (#973)
Scalingo/documentation: _posts/platform/internals/2000-01-01-routing.md

and is reaching the servers with HTTPS, the connection will be upgraded to
HTTP/2 by our routing nodes.

The traffic between the routing nodes and the application containers is then done

> using HTTP/1.1, so your application should **not** expect HTTP/2 traffic.

Reviewer: ```suggestion
using HTTP/1.1. Hence, your application should **not** expect HTTP/2 traffic.
```

Performance improvement due to HTTP/2 would still be present as the protocol

> has been designed to be faster over slow connections (ie. mobile, xDSL) and these

Reviewer: ```suggestion
has been designed to be faster over slow connections (i.e. mobile, xDSL) and these
```

terminals would use HTTP/2. Then, multiplexed requests are achieved in parallel

> using HTTP/1.1 to the containers, as these are in the same network, HTTP/2 would

Reviewer: ```suggestion
using HTTP/1.1 to the containers. As these are in the same network, HTTP/2 would
```
